modern/src/LoginPage.js

```diff
@@ -114,27 +127,42 @@ const LoginPage = () => {
             value={password}
             type='password'
             autoComplete='current-password'
-            onChange={handlePasswordChange} />
-
-          <FormControl fullWidth margin='normal'>
-            <div className={classes.buttons}>
-              <Button type='button' variant='contained' onClick={handleRegister} disabled={!registrationEnabled}>
+            onChange={handlePasswordChange}
+            variant='filled' />
+          <Button 
+            type='submit'
+            variant='contained' 
+            color='secondary'
+            className={classes.submit}
+            disabled={!email || !password}
+            fullWidth>
+            {t('loginLogin')}
+          </Button>
+          <Grid container>
+            <Grid item xs={3}>
+              <Button className={classes.register} onClick={handleRegister} disabled={!registrationEnabled} color="secondary">
                 {t('loginRegister')}
               </Button>
-              <Button type='submit' variant='contained' color='primary' disabled={!email || !password}>
-                {t('loginLogin')}
-              </Button>
-            </div>
-          </FormControl>
+            </Grid>
+            <Grid item xs={9}>
+              <FormControl variant="filled" margin="normal" fullWidth>
+                <InputLabel>{t('loginLanguage')}</InputLabel>
+                <Select>
+                  <MenuItem value="en">English</MenuItem>
```

Comment:
I see some hardcoded text.

Reply:
From where I can list of all languages?

Reply:
I think you'll need to export the list from `common/localization`. But you'll need assistance from @tananaev to implement the entire language switcher

Reply:
We shouldn't use `localization` for it, but we need a list of languages somewhere. This is what we have for the old app:
https://github.com/traccar/traccar-web/blob/master/web/load.js#L39-L91
Maybe we can copy the list somewhere.